nengo_gui/static/data_to_csv.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * A function that retuns simulation data as a csv
+ *
+ * @param {Nengo.Component.components} data_set A list of the graph items in the simulation
+ */
+
+
+var data_to_csv = function(data_set){
+
+  var values = [];
+  var dim_values = [];
+  var times = [];
+  var csv = [];
+  var csv_string = "";
+
+  var data_set = data_set.filter(function(data){
```

Comment:
Maybe add a comment noting that all graphs saving data over time use both of these as constructors.